### PR TITLE
Add Report Issue strings

### PR DESCRIPTION
**Pull request by codemist**
Design specs:
image 
image

### CODE REVIEW

File: en/messages.json [hunk 1]
```diff
@@ -65,6 +65,33 @@
...
+    "popupReportIssueSiteLabelHeadline": {
+        "message": "I’m having issues with this website"
+    },
+    "popupReportIssueSituationHeadline": {
+        "message": "Issue situation"
+    },
+    "popupReportIssueCaseOne": {
+        "message": "Email mask was not accepted by website"
+    },
+    "popupReportIssueCaseTwo": {
```
Comment (peiying2) [inline en/messages.json:83]: Add a comment to say Do Not translate the product name. Check examples of the same instruction in this file so they stay consistent.

**Review by peiying2 — COMMENTED** [review]
@codemist looks good overall. Just one comment on calling out Relay as a product name and shouldn't be translated. There are plenty of mentions like this in this file. Please use the same format and apply that here. Thank you!

File: en/messages.json (hunk 1 above)
Comment (codemist) [reply en/messages.json:83]: Got it! Thanks for the heads up.

**Review by peiying2 — APPROVED** [review]
Looking good! Thank you.

**Pull request merged**